braph2genesis/graphs/_OrdMxBUD.gen.m

```diff
@@ -0,0 +1,203 @@
+%% ¡props_update!
+
+%%% ¡prop!
+NAME (constant, string) is the name of the ordinal binary undirected multiplex with fixed densities.
+%%%% ¡default!
+'OrdMxBUD'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the ordinal binary undirected multiplex with fixed densities.
+%%%% ¡default!
+'In a ordinal binary undirected multiplex with fixed densities (BUD), the layers are those of binary undirected (BU) multiplex graphs derived from the same weighted supra-connectivity matrices binarized at different densities. The supra-connectivity matrix has a number of partitions equal to the number of densities.The layers are connected in an ordinal fashion, where just consecutive layers are connected'
+
+%%% ¡prop!
+TEMPLATE (parameter, item) is the template of the ordinal binary undirected multiplex with fixed densities.
+%%% ¡_prop!
+% % % TEMPLATE (parameter, item) is the graph template to set the graph and measure parameters.
+% % % %%%% ¡_settings!
+% % % 'MultiplexBUD'
+
+%%% ¡prop!
+ID (data, string) is a few-letter code for the ordinal binary undirected multiplex with fixed densities.
+%%%% ¡default!
+'OrdMxBUD ID'
+
+%%% ¡prop!
+LABEL (metadata, string) is an extended label of the ordinal binary undirected multiplex with fixed densities.
+%%%% ¡default!
+'OrdMxBUD label'
+
+%%% ¡prop!
+NOTES (metadata, string) are some specific notes about the ordinal binary undirected multiplex with fixed densities.
+%%%% ¡default!
+'OrdMxBUD notes'
+
+%%% ¡prop!
+GRAPH_TYPE (constant, scalar) returns the graph type __Graph.ORDERED_MULTIPLEX__.
+%%%% ¡default!
+Graph.ORDERED_MULTIPLEX
+
+%%% ¡prop!
+CONNECTIVITY_TYPE (query, smatrix) returns the connectivity type __Graph.BINARY__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.BINARY * ones(layernumber);
+
+%%% ¡prop!
+DIRECTIONALITY_TYPE (query, smatrix) returns the directionality type __Graph.UNDIRECTED__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.UNDIRECTED * ones(layernumber);
+
+%%% ¡prop!
+SELFCONNECTIVITY_TYPE (query, smatrix) returns the self-connectivity type __Graph.NONSELFCONNECTED__ on the diagonal and __Graph.SELFCONNECTED__ off diagonal.
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.SELFCONNECTED * ones(layernumber);
+value(1:layernumber+1:end) = Graph.NONSELFCONNECTED;                
+
+%%% ¡prop!
+NEGATIVITY_TYPE (query, smatrix) returns the negativity type __Graph.NONNEGATIVE__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.NONNEGATIVE * ones(layernumber);
+
+%%% ¡prop!
+A (result, cell) is the cell array containing the multiplex binary adjacency matrices of the binary undirected multiplex. 
```

Comment:
is the cell array containing the multiplex binary **supra**-adjacency matrices of the binary undirected multiplex